eldritch/characters.py

```diff
@@ -569,10 +624,11 @@ def random_possessions(self):
 
 
 def CreateCharacters():
+  from eldritch.expansions.seaside import characters as seaside_characters
```

Comment:
I assume the right way to do this is actually to allow the host to select which expansions to add in, then everything gets added in then, then character selection starts.